docs/dapp/sapphire/README.mdx

```diff
@@ -4,25 +4,29 @@ import {AddSapphireToMetaMask as S, AddSapphireTestnetToMetaMask as ST} from '@s
 
 # Sapphire ParaTime
 
-Sapphire is our official confidential ParaTime providing a smart contract
-development environment with [Ethereum Virtual Machine (EVM)] compatibility.
-
-As the official confidential EVM-compatible ParaTime on the Oasis Network,
-Sapphire allows for:
+Sapphire is our official confidential ParaTime for smart contract development
+with [Ethereum Virtual Machine (EVM)] compatibility.
 
 * Confidential state, end-to-end encryption, confidential randomness
```

Comment:
I'm very conflicted because I see the importance of highlighting these key features, but don't find these as must have for a developer going through Sapphire the very first time.

Reply:
@rube-de

Reply:
I think a quick intro is fine, and then something like a Getting started section with
new to sapphire => checkout the quickstart
want to build on sapphire => get started with our cheatsheet
want to deepdive => advanced stuff in guide
loooking for network information => go to network information page
looking to get test token => faucet